src/export-page/export-stepper/ExportStepper.scss

```diff
@@ -0,0 +1,3 @@
+.pgn__stepper-header-step-list {
+  flex-direction: column;
```

Comment:
Try to use [bootstrap utils](https://getbootstrap.com/docs/4.0/utilities/flex/#direction) for flex-direction declaration